docs/macros-attributes/payable.md

```diff
@@ -15,6 +15,9 @@ An ink! message by default will reject calls that additional fund the smart cont
 Authors of ink! smart contracts can make an ink! message payable by adding the `payable`
 flag to it. An example below:
 
+Note that ink! messages are payable inherently modify the blockchain state, 
```

Comment:
```suggestion
Note that payable ink! messages inherently modify the blockchain state, 
```

Reply:
Argh, had commented, but forgot to commit before merge 🙈